code/kit_version.R

```diff
@@ -0,0 +1,107 @@
+# SET UP
+setwd("C:/Users/kitsc/KIT_code_testing/data")
+library(haven)
+library(tidyverse)
+library(plm)
+library(dplyr)
+
+# DEFINITIONS
+
+##	seed
```

Comment:
It's standard practice to start comments with just the hash symbol and a space, not two hashes. See the style guide I link in my next comment for more tips.